### PR TITLE
[enhance](mtmv)use version instead of timestamp

### PR DESCRIPTION
**Pull request by zddr**
## Proposed changes

Issue Number: close #xxx

MTMV records snapshot information for each refresh of data, used to compare whether partitions need to be updated

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Review by morningman — COMMENTED** [review]
Please add ut for `MTMVRefreshSnapshot` and `MTMTUtil`

File: fe/fe-core/src/main/java/org/apache/doris/mtmv/MTMVMaxTimestampSnapshot.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.mtmv;
+
+import com.google.common.base.Objects;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * The version cannot be obtained from the hive table,
```
[the remark this answers is not in the document]
Comment (zddr) [reply fe/fe-core/src/main/java/org/apache/doris/mtmv/MTMVMaxTimestampSnapshot.java:24]: yes

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by wuwenchi — APPROVED** [review]

**Pull request merged**